app/konnect/runtime-manager/composite-runtime-groups/index.md

```diff
@@ -0,0 +1,111 @@
+Consumer | Credential | By ID
+Consumer | Consumer group | By ID
+Consumer | ACL group | By string
+Consumer groups | Plugin | By string
+Plugin (Non-Global) | Service, route, consumer | By ID
+Global plugin | Runtime Group | By Runtime group
+Key | Key set | By ID
+Vault | Runtime group | By Runtime group
+deGraphQL route | Service | By ID
+GraphQL Rate Limiting cost decoration | Service | By ID
+
+The {{site.base_gateway}} resource associated with an entity must be part of the same standard runtime group as the entity.
+
+Entity-specific behavior exceptions:
+* **Consumers**: A consumer of a standard runtime group becomes a consumer of the composite runtime group once the originating group becomes a member of the composite runtime group.
+The authentication credentials of a consumer in a standard runtime group become valid credentials of the composite runtime group.
+The ID of a consumer from one composite runtime group member can't be used in authorization for another composite runtime group member.
+
+* **Consumer groups**: Only consumers from the same runtime group can be added to a consumer group.
+Consumer groups names in the Rate Limiting Advanced plugin can reference group names from other composite runtime group members.
+
+* **Global plugins**: A plugin that is globally scoped in the standard runtime group remains globally scoped in the composite runtime group. 
+This plugin will affect the entire composite runtime group.
+For example, two instances of the Rate Limiting plugin cannot be installed in the composite runtime group.
+
+* **Vaults**: The prefix of each Vault must be unique.
+Once a Vault from a standard runtime group becomes part of a composite runtime group, it becomes available to the whole composite runtime group.
+An entity field in a standard runtime group can successfully reference a secret in a Vault from another standard runtime group, now both part of the composite runtime group.
+
+## Limitations
+
+A composite runtime group composition will be applied even if the configurations of the standard runtime groups are not combined successfully. 
+This means that even if there is some confict and the member groups weren't merged successfully, a composite runtime group still gets created.
+
+Composite runtime groups are read-only (with some exceptions), so configuration modifications must be made via a member standard runtime group. 
+
+The following are exceptions to the read-only rule:
+* A runtime instance client certificate can be generated in the UI or uploaded to a composite runtime group.
+* Runtime instances can be connected to a composite runtime group, however members of a composite runtime group cannot have any runtime instances connected to them.
+
+Kong Ingress Controller runtime groups can't be part of a composite runtime group.
+
+Analytics for a composite runtime group are only available at the composite level. 
+Member standard runtime groups have no individual analytics reporting.
+
```

Comment:
Can we also add a limitation that conflict detection happens in a CRG only after you have added a runtime instance to the CRG? Thanks!